tests/page_tests.py

```diff
@@ -72,13 +71,13 @@ def replace_selected_specials_with_whitespace(text):
             return altered_text
 
         def strip_pre_tag_blocks(text):
-            return strip_between_tags(r"<pre>|</pre>", text)
+            return strip_between_tags(self, r"<pre>|</pre>", text)
```

Comment:
If these are class functions self should be in the arguments, as at a quick glance I can't see where self comes from, or even know which self it refers to